### PR TITLE
Add installation instructions

### PR DESCRIPTION
**Pull request by jeremiak**
Added some installation instructions so that other folks could get this up and running on their machine. @jenniferthibault do these look similar to the instructions that worked for you earlier today?

Please feel free to edit copy to match desired tone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,20 @@ This is the home of 18F's evolving brand assets and style guide. It's a work in
...
+
+0. `git clone git@github.com:18F/brand.git && cd brand`
+0. `bundle install`
+0. `cd web-design-standards && git submodule init && git submodule update && cd ..`
+0. `bundle exec jekyll serve`
+0. You should be able to see the branding site at [http://127.0.0.1:4000](127.0.0.1:4000)
+
+### Making changes
+Yay, that's exciting that you want to contribute! Follow the installation instructions to get started but add the `--watch` option to the fourth step so you'd run `bundle exec jekyll serve --watch`. Jekyll is now instructed to watch for changes in your site and then rebuild when it notices them. This helps for local development because it means you can save a change and then refresh in your browser to see it.
+
```
Comment (maya) [inline README.md]: @jeremiak what types of changes will `--watch` look for? The reason I ask is I'm only running `jekyll serve` to serve the project and Sass is already re-compiled when I save a change.

**Comment by jenniferthibault** [comment]
@jeremiak thank you for getting this together! @ccostino is going to be excited, because I had just reached out to the documentation working group for help on these instructions!

To get it running yesterday, Carlo and I found that I only needed to do:
`jekyll build`
`jekyll serve` 

and didn't need to `watch` anything

**Comment by ccostino** [comment]
Oh awesome, thank you @jeremiak!  We'd like to make some improvements for the future but this is definitely a great help now! :-)

File: README.md (hunk 1 above)
Comment (jenniferthibault) [inline README.md]: Same for me, should we remove this last bit about `--watch` ?

**Comment by maya** [comment]
@jeremiak @ccostino I don't think you should need to `jekyll build`, `jekyll serve` is all I needed.

**Comment by jenniferthibault** [comment]
I'm going to merge this in as is and we can edit that bit out later if need be. Didn't mean to let this stand so long, thanks!

**Pull request merged**